robots/car_display.h

```diff
@@ -0,0 +1,200 @@
+#pragma once
+
+// BoB robotics includes
+#include "../common/pose.h"
+
+// Third-party includes
+#include "../third_party/units.h"
+
+// SDL
+#include <SDL2/SDL.h>
+
+// Standard C includes
+#include <cmath>
+#include <cstdlib>
+
+// Standard C++ includes
+#include <iostream>
+#include <string>
+#include <utility>
+#include <vector>
+
+namespace BoBRobotics {
+namespace Robots {
+using namespace units::literals;
+
+class CarDisplay
+{
+    using meter_t = units::length::meter_t;
+    using millimeter_t = units::length::millimeter_t;
+    using degree_t = units::angle::degree_t;
+    using degrees_per_second_t = units::angular_velocity::degrees_per_second_t;
+    using meters_per_second_t = units::velocity::meters_per_second_t;
+    using second_t = units::time::second_t;
+
+public:
+    CarDisplay(const millimeter_t screenHeight = 3.2_m, const millimeter_t carWidth = 16.4_cm)
+      : m_MMPerPixel(screenHeight / WindowHeight)
+    {
+        SDL_Init(SDL_INIT_VIDEO);
+
+        m_Window = SDL_CreateWindow("Wheeled-robot simulator",
+                                    SDL_WINDOWPOS_UNDEFINED,
+                                    SDL_WINDOWPOS_UNDEFINED,
+                                    WindowWidth,
+                                    WindowHeight,
+                                    0);
+
+        m_Renderer = SDL_CreateRenderer(m_Window, -1, 0);
```

Comment:
I think if you stick that ``SDL_RENDERER_PRESENTVSYNC`` flag here you shouldn't need to sleep manually - it'll keep the whole loop synchronised to 60fps (16.6ms per update)